cumulus_library/builders/psm_builder.py

```diff
@@ -67,8 +67,9 @@ def __init__(self, toml_config_path: str, data_path: pathlib.Path, **kwargs):
         except OSError:
             sys.exit(f"PSM configuration not found at {self.toml_path}")
         try:
+            toml_dir = pathlib.Path(self.toml_path).parent
             self.config = PsmConfig(
-                classification_json=f"{pathlib.Path(self.toml_path).parent}/{toml_config['classification_json']}",
+                classification_json=str(toml_dir.joinpath(toml_config["classification_json"])),
```

Comment:
Small behavior change here - before, if the toml specified an absolute path, we'd still prepend the toml dir path. Now we support both absolute and relative filenames for this toml field. (I used this in my test, which is why I hit it)